Doc/library/os.rst

```diff
@@ -3780,6 +3780,168 @@ features:
    .. versionadded:: 3.10
 
 
+.. function:: timerfd_create(clockid, flags)
+
+   Create and return a timer file descriptor. The file descriptors supports
+   raw :func:`read` with a buffer size of 8, :func:`~select.select`, :func:`~select.poll`
+
+   .. seealso:: :manpage:`timerfd_create(2)`
+
+
+   Refer to :ref:`time-clock-id-constants` for a list of accepted values for *clockid*.
+
+   *flags* can be constructed from :const:`TFD_NONBLOCK` and :const:`TFD_CLOEXEC`.
+
+   If the timerfd counter is zero and :const:`TFD_NONBLOCK` is not
+   specified, :func:`read` blocks.
+
+   Example::
```

Comment:
These two examples feel out-of-place in the documentation for `timerfd_create` specifically. How about adding a new subheading for timer file descriptions (to include all 5 functions / 4 flags) and put the examples in the introduction to this new subsection?